packages/quantic/force-app/main/default/lwc/quanticTimelineEvent/quanticTimelineEvent.js

```diff
@@ -0,0 +1,128 @@
+import {LightningElement, api} from 'lwc';
+
+/**
+ * The `QuanticTimelineEvent` component displays a single user action event in the user action timeline.
+ * @category Insight Panel
+ * @example
+ * <c-quantic-timeline-event title="Example user action" type="view" timestamp="10:45" search-hub="default"></c-quantic-timeline-event>
+ */
+export default class QuanticTimelineEvent extends LightningElement {
+  /**
+   * The title of the timeline event.
+   * @api
+   * @type {string}
+   */
+  @api title;
+  /**
+   * The type of the timeline event.
+   * @api
+   * @type {`click` | `view` | `custom` | `search` | `case-creation` | `active-case-creation`}
+   */
+  @api type;
+  /**
+   * The timestamp of the timeline event.
+   * @api
+   * @type {string}
+   */
+  @api timestamp;
```

Comment:
I guess you're planning on localizing the timestamp into a string in a parent component? FYI in the headless controller it's likely to be a number

Reply:
Yes that's what I'm planing to do but it's still subject to change, my plan is to prepare the PRs for the user action components and then wait for the Headless user action comptroller before merging the PRs in order to adjust few things, like the one you pointed out in this comment, according to the state and the controller in Headless without having to introduce any breaking change to these new Quantic components.